lib/ansible/executor/task_executor.py

```diff
@@ -1080,7 +1080,7 @@ def start_connection(play_context, variables):
     python = sys.executable
     master, slave = pty.openpty()
     p = subprocess.Popen(
-        [python, ansible_connection, to_text(os.getppid())],
+        [python, ansible_connection, to_text(os.getppid()), to_text(task_uuid)],
```

Comment:
It looks like this should be omitted, since `ansible-connection` in 2.9 doesn't use a second argument.

Reply:
yes, you are correct. This is to support ansible.netcommon collection for ansible 2.9. Something we the content team would like to get support for.

Reply:
I discussed this with @Qalthos. While it isn't used by `ansible-connection` from Ansible 2.9, it is possible to use an alternative version which supports it. Including it here shouldn't cause any issues.